Makefile

```diff
@@ -6,6 +6,7 @@ REPORT_WITH_OFFSET ?= 0
 ENABLE_COVERAGE ?= 1
 COVERAGE_MEMORY_ERRORS ?= 1
 COVERAGE_CONTROL_FLOW_ERRORS ?= 1
+SEED_NON_SPECULATIVE_ERRORS ?= 1
```

Comment:
Our implicit assumption is that the program we fuzz does not include any non-speculative bugs. It doesn't make much sense to fuzz for both speculative and conventional memory violations simultaneously: They are way too different and, I believe, tackling both would make the code base too complicated. 

Anyway, what was the reason you decided to add it? Did you detect any bugs in your benchmarks?

Reply:
Seeding on unexpected error gives you the best-case scenario.
If the program has bugs so you can run the native version on the input and verify it.
If the fuzzing is buggy then you saved the right input to debug it.

My reason was the latter. For traceability of bugs during development.

Reply:
Got it. In this case, could you rename this variable to something more descriptive? Something like STORE_INPUT_ON_SPECFUZZ_FAILURE